test/tablet.py

```diff
@@ -563,11 +563,11 @@ def start_vttablet(
 
     return self.proc
 
-  def wait_for_vttablet_state(self, expected, timeout=60.0, port=None):
+  def _wait_for_vttablet_var(self, var_name, expected, timeout=60.0, port=None):
```

Comment:
It turned out that there's also utils.wait_for_tablet_type() that I can use instead of creating a new function. So I just reverted these changes and modified the test to use utils.wait_for_tablet_type().